backend/lib/io.js

```diff
@@ -324,13 +324,9 @@ function init () {
   setupShootsNamespace(io.of('/shoots'))
   setupJournalsNamespace(io.of('/journals'))
   // start watches
-  _.forEach(watches, (watch, resourceName) => {
-    try {
-      watch(io)
-    } catch (err) {
-      logger.error(`watch ${resourceName} error`, err)
-    }
-  })
+  for (const watch of Object.values(watches)) {
+    watch(io)
```

Comment:
what's the implication of removing the try/catch?

Reply:
If a watch cannot be connected to socket.io the server start should fail and not silently ignore the problem.